conf/nginx.conf

```diff
@@ -1,27 +1,29 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
 location __PATH__/ {
-	alias __FINALPATH__/;
 
-	if ($scheme = http) {
-		rewrite ^ https://$server_name$request_uri? permanent;
-	}
+  # Path to source
+  alias __FINALPATH__/ ;
```

Comment:
I'm thinking that this alias is in fact probably not relevant considerng there's a proxy_pass on that entire location

Reply:
I agree, removed